int32[] freqs
int32 C
---
bool ACK
